mods/measure/init.lua

```diff
@@ -0,0 +1,50 @@
+local function deltas_as_string(pos1, pos2)
+	return("Deltas: X: "..math.abs(pos1.x-pos2.x).." | Y: "..math.abs(pos1.y-pos2.y).." | Z: "..math.abs(pos1.z-pos2.z))
+	end
+
+minetest.register_craftitem("measure:stick", {
+
+	description = "Measuring Stick",
+	inventory_image = "measure_stick.png",
+	stack_max = 1,
+
+	on_use = function(itemstack, user, pointed_thing)
+						       
+		local pos = minetest.get_pointed_thing_position(pointed_thing, above)
```

Comment:
This line should be moved below the `if not pointed_thing or not pos` block

Reply:
If I move the declaration of `pos` below, `if not pointed_thing or not pos` will always evaluate to true.

Reply:
Oh right, good point.